dbcon/dmlpackage/dmltable.cpp

```diff
@@ -72,6 +72,28 @@ int DMLTable::read(messageqcpp::ByteStream& bytestream)
     return retval;
 }
 
+void DMLTable::readMetaData(messageqcpp::ByteStream& bytestream)
+{
+    // read the table name
+    bytestream >> fName;
+
+    // read the schema name
+    bytestream >> fSchema;
+}
+
+void DMLTable::readRowData(messageqcpp::ByteStream& bytestream)
+{
+    messageqcpp::ByteStream::quadbyte rowNum;
+    bytestream >> rowNum;
+
+    for (unsigned int i = 0; i < rowNum; i++)
+    {
+        Row* aRow = new Row();
```

Comment:
where do these get deleted?

Reply:
I see new's but no deletes in this patch.  If the lifecycle of the mem is unclear, feel free to use smart ptrs.

Reply:
You won't see them b/c it happens inside DMLTable dtor so we are relatively safe here. There will be a leak in case of exception though. But this is an issue with the original code that Gagan reused.

Reply:
This is correct. As Roman mentioned, I am reusing the existing code here. The respective deletes are happening in the dtors of InsertDMLPackage, DMLTable and Row objects.